core/libraries/form_sections/strategies/display/EE_Checkbox_Display_Strategy.strategy.php

```diff
@@ -19,16 +19,18 @@ class EE_Checkbox_Display_Strategy extends EE_Compound_Input_Display_Strategy
     public function display()
     {
         $input = $this->get_input();
-        $multi = count($input->options()) > 1;
         $input->set_label_sizes();
         $label_size_class = $input->get_label_size_class();
-        $html             = '';
+        $html = '';
         if (! is_array($input->raw_value()) && $input->raw_value() !== null) {
             EE_Error::doing_it_wrong(
                 'EE_Checkbox_Display_Strategy::display()',
                 sprintf(
-                    esc_html__(
+                    esc_html_x(
                         'Input values for checkboxes should be an array of values, but the value for input "%1$s" is "%2$s". Please verify that the input name is exactly "%3$s"',
```

Comment:
Changing this to a multi-line string breaks the translations, right? So I think its better to leave it as a long string, right?

Reply:
The second argument isn't actually the translation, it's a comment for the translation - so it won't break translations.  However I do think that it'd be good to concatenate the second argument rather than adding line breaks and all the spacing (it'll might look ugly and really hard to follow in whatever client is used for the translation).

Reply:
oh right, ya that's the explanation string. Sure good idea.

Reply:
> What's the benefit of concatenating the second argument?

You mean other than what I already wrote, which was:

> it might look ugly and really hard to follow in whatever client is used for the translation.

In case that isn't clear, what I mean by that, is the formatting of the translation explanation you did might result in odd line-breaks and spacing in the end users client.  I don't know for sure, I just thought that might be the case.  I suppose, however, argument could be made that the script generating the pot might not capture the argument correctly either because it doesn't actually parse php (and thus miss that its concatenated).

So, in the end, it might be better to leave translation explanations as long strings as well.

Reply:
oh ok. I'm inclined to leave this as-is for now, then check out how this string appears on translate.eventespresso.com (the string should appear there as soon as its committed right?)

Reply:
No, it won't appear there as soon as its committed.  Someone on the team (I think usually @Pebblo ) triggers pot builds using grunt and then imports them into the glotpress instance.  _Then_, the strings show up there.

Reply:
> what I mean by that, is the formatting of the translation explanation you did might result in odd line-breaks and spacing in the end users client. 

oooohhhhh  kk.. I thought you were saying that concatenation might look funny, ie: 

```
'Input values for checkboxes should be an array of values, '
 . 'but the value for input "form-input-id" is "form-input-value". '
 . 'Please verify that the input name is exactly "form_input_name[]"',
```

was uglier than 

```
'Input values for checkboxes should be an array of values, 
but the value for input "form-input-id" is "form-input-value". 
Please verify that the input name is exactly "form_input_name[]"',
```

but ya, you mean that what I did could look bonkers in a translation app. Makes sense, will change.

Reply:
yeaaah just making the line really long is my preference, I'm not sure how translation apps will handle line breaks